### PR TITLE
Readme Update with FAQ Section

### PR DESCRIPTION
**Pull request by nosyn00b**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,28 @@ Contains:
 
 4. Rename a new bookmark with right-click
 
+## FAQ
+
+### How can I search for games in subscriptions and owned games at the same time?
```
Comment (UncleGoogle) [inline README.md:83]: Ah, so you mean a general UI trick in Galaxy. I didn't understand firstly.

I'm not convinced to put info like that in this repo, it fits better in awesome-gog-galaxy or a separate FAQ gist to aim bigger audience.

But here we can be more specific, like:

`Can I group my Humble games and Choice games altogether?`

and then the same solution but slightly modified (with additional filter for humble).

File: README.md [hunk 2]
```diff
@@ -78,6 +78,28 @@ Contains:
...
+
+2. Change the filter selecting both owned games and subcription
+
+  ![Filter Configuration](https://user-images.githubusercontent.com/77177410/106529888-88ca8d00-64eb-11eb-96e2-3c55ff4ddc9b.png)
+
+3. Bookmark the search you just composed
+
+  <img src="https://i.gyazo.com/d3cb0876788849a9117f739e3b5d875a.png" width="400">
+
+4. Now you can normally searchthe new  _All games_ view as you do with all bookmarks view
```
Comment (UncleGoogle) [inline README.md:97]:
```suggestion
4. Now you can normally search the new  _All games_ view as you do with all bookmarks view
```

**Review by UncleGoogle — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nosyn00b) [reply README.md:83]: Ok i think i cannot edit my own PR, so I will do another one, you can reject this!

Comment (UncleGoogle) [reply README.md:83]: can't you? And if you edit it on your repo?

Comment (nosyn00b) [reply README.md]: I think I made it :) have a look!

**Comment by nosyn00b** [comment]
Let me know if It's ok now.

File: README.md [hunk 3]
```diff
@@ -66,18 +66,44 @@ Contains:
...
+
+3. The view that you just created will show in the bookmark list as an _Humble Bundle_ duplicate, so you want to rename it. To do so click with right mouse button on the new bookmark itself and select _Rename bookmark_ changing name with something you like as for example _Humble Store & Subsc_.
+
+![Humble Rename Bookmark](resources/Bookmark_rename.png)
+
+4. Now when you select the new bookmark you can view (and search!) both owned and subscription Humble Bundle games.
+
+  ![Hmmble Store and Subsc](resources/Bookmark_renamed.png)
+
+Tip: you can also create an _All games_ bookmark in a similar way to include also subscriptions games as they were owned games.
```
Comment (UncleGoogle) [inline README.md:105]:
```suggestion
Tip: you can also create an _All games_ bookmark for all other platforms in a similar way.
```

**Review by UncleGoogle — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -66,18 +66,44 @@ Contains:
...
-<img src="https://i.gyazo.com/d3cb0876788849a9117f739e3b5d875a.png" width="400">
+![Humble bookmarked](resources/Humble_bookmarked.png)
 
 4. Rename a new bookmark with right-click
 
+## FAQ
+
+### Can I group my Humble games and Choice games altogether?
+
+You have to create a new filtered HUble Bungle bookmark, proceeding as follows:
```
[the remark this answers is not in the document]
Comment (nosyn00b) [reply README.md:85]: :-)

File: README.md [hunk 5]
```diff
@@ -66,18 +66,44 @@ Contains:
...
+
+2. Change the view filter selecting the funnel icon, then the Status menu -> and then flagging both _Owned_ and _Subcriptions_
+
+![HumbleBundle_Filters](resources/HumbleBundle_Filters.png)
+
+3. Bookmark the search query you just composed clicking on the small bookmark flag
+
+![Humble bookmarked](resources/Bookmarking.png)
+
+3. The view that you just created will show in the bookmark list as an _Humble Bundle_ duplicate, so you want to rename it. To do so click with right mouse button on the new bookmark itself and select _Rename bookmark_ changing name with something you like as for example _Humble Store & Subsc_.
```
[the remark this answers is not in the document]
Comment (nosyn00b) [reply README.md:97]: Ok!

File: README.md [hunk 6]
```diff
@@ -66,18 +66,42 @@ Contains:
...
 
 4. Rename a new bookmark with right-click
 
+## FAQ
+
+### Can I group my Humble games and Choice games altogether?
+
+You have to create a new filtered Humble Bundle bookmark, proceeding as follows:
+
+1. Click on the defauit _Humble Bundle_ bookmark: you see only the owned games in this view, and your goals is to have a similar view that lists both owned and subscritptions games.
```
Comment (UncleGoogle) [inline README.md:87]:
```suggestion
1. Click on the default _Humble Bundle_ bookmark: you see only the owned games in this view, and your goal is to have a similar view that lists both owned and subscription games.
```

File: README.md [hunk 7]
```diff
@@ -66,18 +66,42 @@ Contains:
...
+
+2. Change the view filter selecting the funnel icon, then the Status menu -> and then flagging both _Owned_ and _Subcriptions_
```
Comment (UncleGoogle) [inline README.md:89]:
```suggestion
2. Change the view filter selecting the funnel icon, then the Status menu -> and then flagging both _Owned_ and _Subcriptions_.
```

File: README.md [hunk 8]
```diff
@@ -66,18 +66,42 @@ Contains:
...
+
+![HumbleBundle_Filters](resources/HumbleBundle_Filters.png)
+
+3. Bookmark the search query you just composed clicking on the small bookmark flag
```
Comment (UncleGoogle) [inline README.md:93]:
```suggestion
3. Bookmark the search query you've just composed clicking on the small bookmark flag.
```

File: README.md [hunk 9]
```diff
@@ -66,18 +66,42 @@ Contains:
...
+
+![Humble bookmarked](resources/Bookmarking.png)
+
+3. The view that you've just created will show in the bookmark list as an _Humble Bundle_ duplicate, so you want to rename it. To do so click with right mouse button on the new bookmark itself and select _Rename bookmark_ changing name with something you like as for example _Humble Owned + Choices_.
```
Comment (UncleGoogle) [inline README.md:97]:
```suggestion
3. The view that you've just created will appear in the bookmark list as an _Humble Bundle_ duplicate, so you want to rename it. To do so, click with right mouse button on the new bookmark itself and select _Rename bookmark_ changing name with something you like as for example _Humble Owned + Choices_.
```

looks more natural for me (but again: I'm not an Englishman, fell free to ignore if you know better)

File: README.md [hunk 10]
```diff
@@ -66,18 +66,42 @@ Contains:
...
+
+![Humble Rename Bookmark](resources/Bookmark_rename.png)
+
+4. Now when you select the new bookmark you can view (and search!) both owned and subscription Humble Bundle games.
+
+  ![Hmmble Store and Subsc](resources/Bookmark_renamed.png)
```
Comment (UncleGoogle) [inline README.md:103]:
```suggestion
![Hmmble Store and Subsc](resources/Bookmark_renamed.png)
```

**Review by UncleGoogle — CHANGES_REQUESTED** [review]

**Comment by UncleGoogle** [comment]
@nosyn00b we're almost there 😄

**Comment by UncleGoogle** [comment]
What about having github-hosted from an issue but not in the repo itself? https://www.youtube.com/watch?v=nvPOUdz5PL4

So, let me upload them here:
Humble_filters
Humble_Settings
Humble_Install
Humble_Groupby
Humble_Bookmarked
HumbleBundle_Filters
Bookmarking
Bookmark_renamed
Bookmark_rename

now we can use those links in readme and save some kb in the repo :)

**Comment by nosyn00b** [comment]
Done!

**Comment by UncleGoogle** [comment]
Thanks!

**Pull request merged**